Source/settings/maps/Pallet Town.js

```diff
@@ -107,11 +112,13 @@ FullScreenPokemon.FullScreenPokemon.settings.maps.library["Pallet Town"] = {
                 { "thing": "DirtLight", "x": 80, "y": 120, "width": 48, "height": 16 },
                 { "thing": "FenceWide", "x": 64, "y": 128, "width": 88 },
                 { "thing": "FenceWide", "x": 8, "y": 128 },
-                { "thing": "AreaSpawner", "y": 128, "map": "Route 21", "area": "Land", "direction": 2 }
+                { "thing": "AreaSpawner", "y": 128, "map": "Route 21", "area": "Land", "direction": 2 },
+                { "thing": "DirtWhite", "x": 40, "y": 40, "width": 16, "entrance": "Fly Location"}
```

Comment:
This should be hidden. Does using a `HiddenTransporter` work? If that fails, is there a `Blank` object you can use? If not, can you add a `@todo` to add one?